roles/rust-foundation-project-director.md

```diff
@@ -26,8 +33,27 @@ Please refer to [Rust Foundation Director Roles & Responsibilities][director-rol
 - A keen sense of the needs of Rust Project, its teams, processes, and ecosystem.
 - Willingness to understand and represent viewpoints they may not have personal experience with or necessarily agree with.
 
+## Requirements and Eligibility
+
+There are requirements and eligibility criteria Project Directors must meet.
+These are set by the Rust Project, the [Foundation bylaws], the laws of the State of Delaware, and the laws of the United States of America.
+
+- Project directors must be members of the Rust Project in good standing.
+  Among other things, this means they must not be currently under any moderation sanction.
+- Project directors must disclose their name on the Foundation's IRS Form 990, which is a public document. For an example, see [the Foundations 2021 filing][form-990].
+- A candidate may not be elected as a Project Director if they are employed by a corporate member of the Foundation that already employs another Project Director.
```

Comment:
```suggestion
- No two project directors can be employed by the same corporate member of the Foundation.
```

Maybe we can simplify this language to something like this? I think this is essentially the same language. I imagine that from a Rust project perspective, we may not care about the "corporate member" perspective -- we'd probably want this to just be "any company", but that can be discussed separately.

Reply:
The wording update seems reasonable to me.

My reading of the bylaws is that if some company was not a corporate member then all of the project directors could be from the same company. Is that corrent? If so, is that intended?

I could imagine we want to make it "any company" rather than "any corporate member," but if we do, I think we should make clear how much of that restriction is from the bylaws and how much is the Project adding their own requirements on top of that.